### PR TITLE
changes in act_table, text_table and update of data_dict

### PR DESCRIPTION
**Pull request by inamoto4**
I would recommend to keep ONE commentary/discussion field respectively for the act_table and text_table, but limit the number of its sentences and encourage everybody to focus on the essentials and leave out the rest. Updated data_dict as well.

#84

### CODE REVIEW

File: data/Text.csv [hunk 1]
```diff
@@ -61,14 +61,14 @@ T00044,The Seventies,,en,P0009,G03,PL0002,2009,Sanlian shudian,,,NULL,,,,,,2017-
 T00044,七十年代,,zh,P0009,G03,PL0002,2009,Sanlian shudian,,,NULL,,,,,,2017-08-07,MG,2017-08-11,MG
 T00045,A sunken holy place,,en,P0135,G02,PL0054,1999,Xinjiang qingshaonian chubanshe,,,NULL,,,,,,2017-08-11,MG,2017-08-11,MG
 T00045,沉沦的圣殿,,zh,P0135,G02,PL0054,1999,Xinjiang qingshaonian chubanshe,,,NULL,,,,,,2017-08-11,MG,2017-08-11,MG
-T00050,,,en,P0082,G04,PL0002,,,,,,T00045,,24,THE NOVEL CANNOT BE FOUND,,2017-08-20,MG,2017-10-02,DP
-T00050,消失在暗淡的夜雾中,,zh,P0082,G04,PL0002,,,,,,T00045,,24,recommended by Guo Shiyin / It's about a man left his father and devoted himself in dangerous work  the reader said it is similar to his situation.,,2017-08-20,MG,2017-08-20,MG
+T00050,,,en,P0082,G04,PL0002,,,,,,T00045,,24,,THE NOVEL CANNOT BE FOUND,2017-08-20,MG,2017-10-02,DP
+T00050,消失在暗淡的夜雾中,,zh,P0082,G04,PL0002,,,,,,T00045,,24,recommended by Guo Shiyin / It's about a man left his father and devoted himself in dangerous work the reader said it is similar to his situation,what does „recommended“ mean here?,2017-08-20,MG,2017-08-20,MG
```
Comment (duncdrum) [inline data/Text.csv]: avoid German up-down style quotation marks. Use only ``"`` or ``'``, not ``‟`` they break stuff when rendering to html

File: data/helper/data_dictionary.csv [hunk 2]
```diff
@@ -30,16 +31,15 @@ SocialPosition,last_modified_by,acronym of project member last modifying this li
 ,,,,,,,
 Text,text_id,T and a five-digit number (leading),,2017-07-04,LH,2017-07-04,LH
 Text,title,Title of a text. For autobiographical discussions of reading texts both : a) title in Chinese characters; b) title in original language,,2017-07-04,LH,2017-07-04,LH
-Text,title_lang,Language code of this title ('en' or 'zh'),,2017-07-04,LH,2017-07-04,LH
+Text,title_lang,"Language code of this title ('en' or 'zh' or 'ru' etc.), in the case of russian publications add three lines of title information",,2017-07-04,LH,2017-07-04,LH
 Text,genre_id,Refer to genre IS,,2017-07-04,LH,2017-07-04,LH
-Text,publication_place,place of first publication,,2017-07-04,LH,2017-07-04,LH
-Text,publication_date,date of first publication in original language,,2017-07-04,LH,2017-07-04,LH
-Text,publishing_house,name of publishing house (hanzi + pinyin),,2017-07-04,LH,2017-07-04,LH
+Text,publication_place  ,"place of publication, first mainland edition between the years 1949 and 1978, after 1978 the edition at hand is sufficient",,2017-07-04,LH,2017-10-27,OS
```
Comment (duncdrum) [inline data/helper/data_dictionary.csv]: see death-by-comma

**Comment by inamoto4** [comment]
Thx for remininding me about the death by (and logically following: "of") comma.
I did the necessary changes.

I am not too sure what DD refers to (data definition?), but I have just kept the names, not added them. I tried to copy them into the note field. I guess we should now just handle the respective entries accordingly and delete the names after that. 

What do you mean exactly by one entry - one sentence? Not one cell - one sentence, right?

Thx again!

**Comment by duncdrum** [comment]
DD = data dictionary. The definition column should have one sentence, that helps people that are not part of this project to understand what's going on. Think about questions that complete stranger smight have, when they read and understand the column name.
### good
| column | definition | note |
| --- | ---| --- |
| pub_year | Publication year given on title page translated into gregorian Iso notation. | @hiwis please make sure to convert taiwan years | 

### Less good
| column | definition | note |
| --- | ---| --- |
| pub_year | There are many different years of publication, you can use 1960, 80AD, 0080, please check this carefully and report back any questions on github. | | 
| pub_year | Year of publication. | |

**Comment by duncdrum** [comment]
not sure what names you plan to delete, or why?

**Pull request merged**